packages/sdk/src/utils/helpers.ts

```diff
@@ -0,0 +1,33 @@
+import md5 from 'crypto-js/md5';
+import { BigNumber, utils } from 'ethers';
+import { BaseTrade } from './types';
+import { getTokenInfo } from './constants';
+
+// would be good to turn prices into ticks to decrease chances of collision
+export async function createAccountId(
```

Comment:
as far as I see, isLp is mandatory here rather than optional?

Reply:
that's true, I made it mandatory and throwing an error now if price range does is missing and isLP=true [c36bf33](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/10/commits/c36bf330f7daf2dab92b10c2d0c1ef4b5e4e1cc0)